apps/src/music/player/ToneJSPlayer.ts

```diff
@@ -405,6 +407,8 @@ class ToneJSPlayer implements AudioPlayer {
   }
 
   private generateEffectBusses() {
+    // Dispose of all existing effect busses
+    Object.values(this.effectBusses).forEach(node => node.dispose());
```

Comment:
Related?

Reply:
Not strictly, but a small cleanup to make sure we're disposing of existing effect busses before creating new ones. Can break this out into a separate change if it would be safer?

Reply:
I think it's fine to include here.